docs-web/configuration/security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+We consider the security of the Ingress Controller paramount to the success of our users and use the following to 
+ensure the security of the Ingress Controller:
+* [Secure Development Life Cycle](https://www.microsoft.com/en-us/securityengineering/sdl/)
+* [FOSSA](https://fossa.com) scanning
+
+However, the Ingress Controller is deployed by a User in their environment, and as such, the User takes responsibility 
+for securing a *deployment* of the Ingress Controller. 
+We highly recommend every User to read and understand the following security concerns.
```

Comment:
I would change this to "We strongly recommend every User read and understand...."
Rationale:
- "recommend to" is awkward imho/
- "highly recommend" is generally used for positive experiences like recommending a show. "strongly recommend" is more suitable for instructions that should be followed imho.

Reply:
updated